x/gov/genesis.go

```diff
@@ -15,6 +15,7 @@ func InitGenesis(ctx sdk.Context, ak types.AccountKeeper, bk types.BankKeeper, k
 	k.SetDepositParams(ctx, *data.DepositParams)
 	k.SetVotingParams(ctx, *data.VotingParams)
 	k.SetTallyParams(ctx, *data.TallyParams)
+	k.SetBurnParams(ctx, *data.BurnParams)
```

Comment:
let's add a state-machine breaking change for this.

Reply:
this was removed, merged with deposit